internal/p2p_test.go

```diff
@@ -288,3 +288,7 @@ func TestApplyBlockFailure(t *testing.T) {
 		t.Errorf("Incorrect number of blocks applied, expected %d, got %d", expectedBlocksApplied, len(sendRPC.BlocksApplied))
 	}
 }
+
+func ErrorHandlerTest(t *testing.T) {
+
+}
```

Comment:
Empty function

Reply:
I cleaned this up later in 670a471e01ccd2fed4f5afe6ea5be03ec53f717b

To prevent future merge conflicts, can this be overlooked?